src/game/AI/ScriptDevAI/scripts/world/npcs_special.cpp

```diff
@@ -2820,6 +2820,110 @@ bool GossipHello_npc_gossip_npc(Player* player, Creature* creature)
     return true;
 }
 
+bool GossipHello_npc_paymaster(Player* pPlayer, Creature* pCreature)
```

Comment:
I do not understand why the hello part is not in DB.

Reply:
I don't know if there are broadcast texts for it etc.